src/lib/cache/subCache.cpp

```diff
@@ -33,6 +33,7 @@
 
```

Comment:
About performance ... I'm afraid we will not gain but lose a little.
The only operation that will benefit from the new implementation should be
`GET /v2/subscriptions/{SUB-ID}`.